### ISSUE TITLE
Add to documentation. How to setup 2 factor authentication.

### ISSUE THREAD

**Issue by chris001** [salesagility/SuiteCRM#3219]
#### Issue

Add to documentation, how to setup 2 factor authentication.

#### Possible Fix

https://netknights.it/en/two-factor-authentication-with-privacyidea-ldap-proxy/

#### Context

2 Factor Authentication is required in most higher security environments. Banking, Government, Health, etc.

**Comment by samus-aran** [comment]
Thanks @chris001 We'll look at updating the User Guide as there are still areas that need to be included.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add to documentation. How to setup 2 factor authentication.](https://trello.com/c/F5kP56UH/18-add-to-documentation-how-to-setup-2-factor-authentication)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
